tools/feature_version_audit.py

```
@@ -0,0 +1,684 @@
+    feature_actions = build_feature_actions(bump_suggestions, metadata_issues, new_features, get_commit_author, normalize_name)
+    # Build author stats directly from actionable suggestions
+    author_stats = build_author_stats(feature_actions)
+    author_stats_lines = format_author_stats(author_stats)
+    actionable_lines = format_actionable_lines(feature_actions)
+    output = "\n".join(lines)
+    output += "\n" + "\n".join(author_stats_lines)
+    output += "\n" + "\n".join(actionable_lines)
+    return output
+
+def main():
+    global RELEASE_TAG, FEATURES_DIR, SHADER_TYPES
```

Flake8 (7.2.0)

[error] `global RELEASE_TAG` is unused: name is never assigned in scope

(F824)